Updated tests to the java VG example

Artem-applitools opened the pull request:
Updated dependencies and readme

AnewTranduil committed:
Updated tests to the java VG example
Updated dependencies and readme

AnewTranduil committed:
updated after hook
Added comments

AnewTranduil committed:
lock file

AnewTranduil committed:
comment updates

Artem-applitools commented:
Updated comments and dependencies to the latest

SophieDePaula commented:
Please add all the configs as here (**replace edge with chromium edge**):

        config.addBrowser(1200, 700, BrowserType.CHROME);
        config.addBrowser(1200, 700, BrowserType.FIREFOX);
        config.addBrowser(1200, 700, BrowserType.EDGE);
        config.addBrowser(768, 700, BrowserType.CHROME);
        config.addBrowser(768, 700, BrowserType.FIREFOX);
        config.addBrowser(768, 700, BrowserType.EDGE);

        config.addDeviceEmulation(DeviceName.iPhone_X, ScreenOrientation.PORTRAIT);
        config.addDeviceEmulation(DeviceName.Pixel_2, ScreenOrientation.PORTRAIT);

and add it to the config file instead in the test method.
Also set the batch and the api key on applitools.config.js

Thanks

AnewTranduil committed:
moved to the config file apiKey, batchName and browsers setups
testcafe SDK doesn't exports Configuration class so config couldn't be like this:
 config.addBrowser(1200, 700, BrowserType.CHROME);
 config.addBrowser(1200, 700, BrowserType.FIREFOX);
 config.addBrowser(1200, 700, BrowserType.EDGE);
 config.addBrowser(768, 700, BrowserType.CHROME);
 config.addBrowser(768, 700, BrowserType.FIREFOX);
 config.addBrowser(768, 700, BrowserType.EDGE);

 config.addDeviceEmulation(DeviceName.iPhone_X, ScreenOrientation.PORTRAIT);
 config.addDeviceEmulation(DeviceName.Pixel_2, ScreenOrientation.PORTRAIT);
